app/src/layouts/PageHeader.vue

```diff
@@ -168,10 +173,13 @@ const showBack = computed(() => !route.meta.rootPage || !store.getters.drawerExi
  * Show the menu button.
  */
 const showMenu = computed(() => !showBack.value && !store.state.ui.drawerPersistent)
+
+const showProfile = props.profile;
```

Comment:
We can use the `route.meta.rootPage` to know if this is a root page, and show the profile in this case. So we don't need to manually set it for each page. This `rootPage` prop is also used for other behaviors (such as if we need to show a back button or not).

If we need to add more customization on top of that, we can then add the prop and override the default if necessary, nut I don't think we actually need it.